packages/gatsby/src/schema/print.ts

```diff
@@ -177,134 +212,127 @@ const printType = (tc, typeName) => {
     return printScalarType(tc)
   } else if (tc instanceof InputTypeComposer) {
     return printInputObjectType(tc)
-  } else {
-    throw new Error(`Did not recognize type of ${typeName}.`)
```

Comment:
TS was complaining, because all possible types in `NamedTypeComposer` were handled. Also we never actually passed `typeName` to this function